webpack/assets/javascripts/react_app/components/charts/Chart.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+    constructor(props) {
```

Comment:
javascript classes do not automatically call parent constructor. The class object is only syntactical sugar.